public/js/AppSorting.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable react/jsx-no-undef */
+/* eslint-disable no-undef */
+
+const sortingAlpha = (symbols, direction) =>
+  _.orderBy(
+    symbols,
+    s => {
+      return s.symbol;
+    },
+    direction
+  );
+
+const sortingSellProfit = (symbols, direction) =>
+  _.orderBy(
+    symbols,
+    s => {
+      return s.sell.currentProfit;
```

Comment:
I think `s.sell.currentProfit` treated as string and sorting doesn't work as expected for this. Putting `+` in front of it will solve the issue.

Reply:
Updated.